example/android/javalib/3-linting/build.mill

```diff
@@ -0,0 +1,95 @@
+// == Linting with Basic Config
+
+// This Mill build configuration includes a linting step, which is essential for ensuring code
+// quality and adhering to best practices in Android development projects. Running the `androidLintRun` task
+// produces a detailed HTML report by default, identifying potential issues in the code, such as performance,
+// security, and usability warnings. This helps maintain the health and quality of the codebase.
+
+// In this example, we set a custom config to generate the report file in txt format and not HTML.
+
+//// SNIPPET:BUILD
+package build
+
+import mill._
+import mill.javalib.android.{AndroidSdkModule, AndroidLintModule}
+
+// Create and configure an Android SDK module to manage Android SDK paths and tools.
+object androidSdkModule0 extends AndroidSdkModule {
+  def buildToolsVersion = "35.0.0"
+  def bundleToolVersion = "1.17.2"
+}
+
+// Actual android application with linting config
+object app extends AndroidLintModule {
+  def androidSdkModule = mill.define.ModuleRef(androidSdkModule0)
+
+  // Set path to the custom `lint.xml` config file. It is usually at the root of the project
+  def androidLintConfigPath = Task { Some(PathRef(millSourcePath / "lint.xml")) }
+
+  // Set the linting report to be generated as a text file
+  def androidLintReportFmt = Task { "txt" }
+
+  // Set path to generated `.jar` files in this case (or `.class` files)
+  def classPath = Task.Source { androidJar() }
```

Comment:
what is the need to dedicated `classPath` which adds only a confusion, because there is already `compileClasspath` and `runClasspath` available and which already include everything what is needed to build the sources?